src/CachePlugin.php

```diff
@@ -87,8 +114,17 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
                     $response = $response->withBody($this->streamFactory->createStream($body));
                 }
 
-                $cacheItem->set(['response' => $response, 'body' => $body])
-                    ->expiresAfter($this->getMaxAge($response));
+                $maxAge = $this->getMaxAge($response);
+                $currentTime = time();
+                $cacheItem
+                    ->expiresAfter($this->config['cache_lifetime'] + $maxAge)
```

Comment:
We would like to serve Responses that the server once said should expire. 

See https://github.com/php-http/cache-plugin/pull/8#issuecomment-236175746